source/standalone/tutorials/04_sensors/benchmark_cameras.py

```diff
@@ -0,0 +1,621 @@
+    default=False,
+    required=False,
+    help="Whether to save clouds as .npys in addition to figures (visualize should also be on)",
+)
+
+parser.add_argument(
+    "--num_tiled_cameras",
+    type=int,
+    default=2,
+    required=False,
+    help="How many tiled cameras to create",
+)
+
+parser.add_argument(
+    "--num_standard_cameras", type=int, default=1, required=False, help="How many normal cameras to create"
+)
+
+parser.add_argument(
+    "--num_ray_caster_cameras", type=int, default=1, required=False, help="How many normal cameras to create"
+)
+
+
+parser.add_argument(
+    "--tiled_camera_replicators",
+    nargs="+",
+    type=str,
+    default=["rgb", "depth"],
+    help="What replicators to use for the tiled camera",
+)
+
+parser.add_argument(
+    "--standard_camera_replicators",
+    nargs="+",
+    type=str,
+    default=["rgb", "distance_to_image_plane"],
+    help="What replicators to use for the usd camera",
+)
+
+parser.add_argument(
+    "--ray_caster_camera_replicators",
+    nargs="+",
+    type=str,
+    default=["distance_to_image_plane"],
+    help="What replicators to use for the ray caster camera",
+)
+
+parser.add_argument(
+    "--ray_caster_visible_mesh_prim_paths",
+    nargs="+",
+    type=str,
+    default=["/World/ground"],
+    help="WARNING: Ray Caster can currently only cast against a single, static, object",
+)
+
+parser.add_argument(
+    "--convert_depth_to_camera_to_image_plane",
+    action="store_true",
+    default=True,
+    help=(
+        "Enable undistorting from perspective view (distance to camera replicator)"
+        "to orthogonal view (distance to plane replicator) for depth."
+    ),
+)
+
+parser.add_argument(
+    "--keep_raw_depth",
+    dest="convert_depth_to_camera_to_image_plane",
+    action="store_false",
+    help=(
+        "Disable undistorting from perspective view (distance to camera)"
+        "to orthogonal view (distance to plane replicator) for depth."
+    ),
+)
+
+parser.add_argument(
+    "--height",
+    type=int,
+    default=120,
+    required=False,
+    help="Height in pixels of cameras",
+)
+
+parser.add_argument(
+    "--width",
+    type=int,
+    default=140,
+    required=False,
+    help="Width in pixels of cameras",
+)
+
+parser.add_argument(
+    "--warm_start_length",
+    type=int,
+    default=3,
+    required=False,
+    help="How many steps to run the sim before starting benchmark",
+)
+
+parser.add_argument(
+    "--num_objects", type=int, default=10, required=False, help="How many objects to spawn into the scene."
+)
+
+parser.add_argument(
+    "--experiment_length",
+    type=int,
+    default=30,
+    required=False,
+    help="How many steps to average over",
+)
+
+AppLauncher.add_app_launcher_args(parser)
+args_cli = parser.parse_args()
+args_cli.enable_cameras = True
+
+if args_cli.visualize:
+    print("[WARNING]: You have selected to visualize. which means your benchmark results will not be meaningful.")
+    import matplotlib
```

Comment:
If you follow the suggestion above to move arg parsing to a function, you can always import matplotlib below, and then do the backend change within main